extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -93,6 +101,22 @@ object PrivilegesBuilder {
       }
     }
 
+    def mergeProjectionWithIdentifier(
+        table: Identifier,
+        cols: Seq[Attribute],
```

Comment:
can be removed

Reply:
removed and reuse plan for column output listing. thx for your code sample.